oluwajuwon/Fast-Food-Fast-Frontend
# #163629305 Users should be able to delete orders

#### What does this PR do?

Adds functionality for users to be able to delete their orders

#### Description of Task to be completed?
 - Write tests for components, actions and reducers
 - Create necessary components
 - Create actions and reducers
 - Boost test coverage

#### How should this be manually tested?

- clone the repo and `cd` into it
- run `npm i` to install project dependencies
- rename the `.env.sample` file to `.env` and provide the required details
- run `npm run start:dev` to start up the application
- sign up for an account at `/signup` or login at `/login`
- navigate to `/my-orders` to view all recent orders made
- Click on delete to delete any of the orders
- Click on yes to confirm delete

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories? (If applicable)

[#163629305](https://www.pivotaltracker.com/story/show/163629305)

#### Screenshots (If applicable)
<img width="1440" alt="screen shot 2019-02-05 at 10 38 06 pm" src="https://user-images.githubusercontent.com/29441898/52305955-c5b02780-2996-11e9-9a14-5b26a2e1ae02.png">
<img width="1440" alt="screen shot 2019-02-05 at 10 38 16 pm" src="https://user-images.githubusercontent.com/29441898/52305956-c5b02780-2996-11e9-83ed-b8244410290e.png">
